src/citrine/resources/project.py

```diff
@@ -221,6 +221,34 @@ def share(self,
             "resource": {"type": resource_type, "id": resource_id}
         })
 
+    def transfer_resource(self, receiving_project_uid: Union[str, UUID],
+                          resource: Resource) -> bool:
+        """
+        Transfer ownership of a resource.
+
+        The new owner of the the supplied resource becomes the project
+        with uid = receiving_project_uid.
```

Comment:
What access does the user need in order to make this call?  E.g. "You must be a lead in the originating project and have write access to the receiving project."

Reply:
At the moment, you must be a lead of both projects, but we are looking into allowing the lead of one project transfer a resource to a project in which the user is a member.